api/v1/views/places.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.city import City
+
+
+@app_views.route('/cities/<city_id>/places',
+                 methods=['GET'], strict_slashes=False)
+def get_places_cities_id(city_id):
```

Comment:
saw in other scripts that this definition could use a better name. Something like "get_cities_id_places"

Reply:
Change fixed